src/main/java/jpass/xml/bind/Entry.java

```diff
@@ -38,6 +42,13 @@
     protected String user;
     protected String password;
     protected String notes;
+    protected String lastModification;
+    protected String creationDate;
+
+    public Entry(){
+        this.creationDate = new SimpleDateFormat("dd-MM-yyyy HH:mm:ss").format(new Date());
```

Comment:
It would be nice if the date format could be configurable via `jpass.util.Configuration`.

Reply:
Yes @gaborbata, I think too. I made some changes, creating the Date as Unix timestamp and rendering it accordingly to the chosen format, using by default the one above. The new key value for the date format is `default.date.format` and it basically follows the format used by Java SimpleDateFormat.